.circleci/config.yml

```diff
@@ -45,6 +45,7 @@ jobs:
             helm init --service-account default --wait
       - run: .circleci/helm-test.sh oathkeeper
       - run: .circleci/helm-test.sh hydra
+      # - run: .circleci/helm-test.sh maester
```

Comment:
Add it for testing? :)

Reply:
Sadly there is nothing to test yet, as we don't have a webserver or anything else to check readiness. I left this commented as a reminder :)

Reply:
I see, that makes sense then!